src/chain-wallet-libs/bindings/wallet-js/src/lib.rs

```diff
@@ -1,13 +1,26 @@
 //! JavaScript and TypeScript bindings for the Jormungandr wallet SDK.
 
-use rand_chacha::rand_core::SeedableRng;
-use rand_chacha::ChaCha20Rng;
-use std::convert::TryInto;
+pub use certificates::Certificate;
+pub use certificates::{
+    vote_cast::{Payload, VoteCast},
+    vote_plan::VotePlanId,
+};
+use chain_core::packer::Codec;
+use chain_core::property::DeserializeFromSlice;
+use chain_impl_mockchain::block::Block;
+pub use fragment::{Fragment, FragmentId};
+pub use utils::set_panic_hook;
 use wasm_bindgen::prelude::*;
 
+mod certificates;
+mod fragment;
 mod utils;
 
-pub use utils::set_panic_hook;
+// When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
+// allocator.
+#[cfg(feature = "wee_alloc")]
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
```

Comment:
I'm not sure we should be using wee_alloc, it's unmaintained and currently has a pretty nasty memory leak issue: https://github.com/rustwasm/wee_alloc/issues/106

I'd vote to just drop wee_alloc support alltogether